src/checkpoint/checkpoint_txn.c

```diff
@@ -2057,7 +2057,7 @@ __checkpoint_lock_dirty_tree(
         }
 
         /* Skip the clean btree until the btree has obsolete pages. */
```

Comment:
this comment needs updating

Reply:
Fixed in a48be7748ff83a92f9440a5d7d7891e665d8c804